Library/Homebrew/cmd/help.rb

```diff
@@ -15,7 +15,7 @@
 
 Developers:
   brew create [URL [--no-fetch]]
-  brew edit [FORMULA...]
+  brew (where|edit) [FORMULA...]
```

Comment:
actually here it is in plain site in help.rb:
```
# NOTE Keep the lenth of vanilla --help less than 25 lines!
```
so how about:
* option A: brew (formula|edit) [FORMULA...]
* option B: brew formula [FORMULA...]
and remove brew edit (which, again, is superseded by `brew formula`, whereas the other direction is not the case)